packages/protocol/test/governance/validators.ts

```diff
@@ -2059,4 +2059,21 @@ contract('Validators', (accounts: string[]) => {
       })
     })
   })
+
+  describe('#forceDeaffiliate', () => {
+    // TODO(lucas): instantiate governace/slasher contracts to test modifier
```

Comment:
You don't actually need to do this, instead, you can register the address that you're making the `forceDeaffiliate` call with as one of the slashers.

There should be examples elsewhere in the tests of how to do this, but it's pretty straightforward